front-end/src/components/Map/LocationViewMap.js

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import { Wrapper } from "@googlemaps/react-wrapper";
+import TableMap from "./TableMap";
+import TableData from "../Tables/TableData";
+import Marker from "./Marker";
+import TablePopup from "./TablePopup";
+import Button from "@mui/material/Button";
+import { CircularProgress } from "@mui/material";
+import axios from "axios";
+
+const render = () => {
+  return (
+    <CircularProgress size="100px" />
+  );
+};
+
+const getTablePosition = tbl => {
+  const coords = tbl.coordinateLocation.coordinates;
+
+  return {
```

Comment:
Per discussion, make sure we add some consts here for Latitude and Longitude (If necessary, per-service)

Reply:
updated with 03f466683b3e75b6c5a78bf46ac051679e7d2dfe